### DISCUSSION TITLE
主页 - Ultralytics YOLOv8 文档

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#6397] [Docs]
# 主页 - Ultralytics YOLOv8 文档

探索Ultralytics YOLOv8的完整指南，这是一个高速、高精度的目标检测和图像分割模型。包括安装、预测、训练教程等。

https://docs.ultralytics.com/zh/

**Comment by zsc14752** [comment]
How can I load a trained model for real-time object detection from a camera? My program keeps encountering an error at the line model = torch.hub.load('ultralytics/yolov5', 'yolov5s', 'custom', path=weights_path, source='local').

**Reply by pderrenger** [reply]
@zsc14752 it looks like you're trying to load a model using `torch.hub.load` which is typically used for YOLOv5 models. For YOLOv8, you should use the `ultralytics` Python package to load your trained model. Here's how you can do it:

```python
from ultralytics import YOLO

# Load your custom trained model
model = YOLO('path/to/your/trained_model.pt')

# Start real-time detection (0 for webcam)
results = model.predict(source=0)
```

Make sure that `'path/to/your/trained_model.pt'` is replaced with the actual path to your trained YOLOv8 model weights file. This should set you up for real-time object detection using your camera. If you encounter any further issues, please check the error message and refer to the [Predict documentation](https://docs.ultralytics.com/modes/predict) for additional guidance. 😊👍
